### ISSUE TITLE
What is Solid?

### ISSUE THREAD

**Issue by jeff-zucker** [solid/solidproject.org#576]
With the advent of the DEIT, it is perhaps worth revisiting the question of how we present what Solid is.  Here is my take.  

**Solid is**
 
  * **a vision** of an inclusive web in which everyone has control of their own data
  * **a technology** defining protocols and specifications for the new web
  * **an ecosystem** of servers, apps, and libraries implementing the protocols
  * **a community** of people working to further the vision and build the new web

**Comment by kay-kim** [comment]
@jeff-zucker  -- I think there's a Solid Team meeting next week.  Could we put this on the agenda because it would be great to make progress on this as I agree that it would be great to define in the various context.

**Comment by csarven** [comment]
This issue is still very much relevant. Will you give a pass at the content? I'm not crazy about pitching it as the "new web" because Solid is supposed to extent the current web (as opposed to rewrite) or specify more clearly/specifically how we move towards the vision, enable the necessary social space... based on our shared principles and values.

**Comment by jeff-zucker** [comment]
> This issue is still very much relevant. Will you give a pass at the content? I'm not crazy about pitching it as the "new web" because Solid is supposed to extent the current web (as opposed to rewrite) or specify more clearly/specifically how we move towards the vision, enable the necessary social space... based on our shared principles and values.

Yes, I was already working on this in relation to the website review.  I find the whole Web layer explanation inappropriately technical here - we want a much more inclusive statement that invites in an audience who may or may not be interested in such things as layers of the web.  Of course it's the underpinning, but it's not why I, for one, got involved in Solid and shouldn't be the main way we pitch it.  I'll post an alternative wording as soon as I can.

**Comment by jeff-zucker** [comment]
> I'm not crazy about pitching it as the "new web" 

Agree, neither am I anymore.

**Comment by jeff-zucker** [comment]
Taking into account @csarven 's objection about pitching Solid as a "new web", I propose this : 
###  Solid is                                                                                     
                                                                                             
* a vision of the web we want in which everyone has control of their own data                
* an architecture defining protocols and specifications for the web we want                  
* an ecosystem of servers, apps, and libraries implementing the protocols                    
* a community of people working to further the vision
